pom.xml

```diff
@@ -186,6 +186,11 @@
                 <artifactId>httpclient</artifactId>
                 <version>4.5.2</version>
             </dependency>
+           
+            <dependency>
+			    <groupId>org.dom4j</groupId>
+			    <artifactId>dom4j</artifactId>
+           </dependency>
```

Comment:
This is fine to keep in, but please consistently use on spaces for indentation here.

Reply:
i think it looks ok now